### PR TITLE
CLOUD-625 - Add chaos-mesh e2e-tests

### PR DESCRIPTION
**Pull request by tplavcic**
[CLOUD-625](https://jira.percona.com/browse/CLOUD-625) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | passed |
| haproxy | passed |
| upgrade-haproxy | passed |
| storage | passed |
| recreate | passed |
| big-data | passed |
| scaling | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| limits | passed |
| self-healing-chaos | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| smart-update | passed |
| affinity | passed |
| self-healing-advanced | passed |
| one-pod | passed |
| auto-tuning | passed |
| self-healing-advanced-chaos | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | failed |
| operator-self-healing | passed |
| upgrade-consistency | passed |
| operator-self-healing-chaos | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/881/commits/d20897214316a9bf781511e440704f5f2804bdb4
image: `perconalab/percona-xtradb-cluster-operator:PR-881-d2089721`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| scaling | passed |
| big-data | passed |
| storage | passed |
| scheduled-backup | passed |
| haproxy | passed |
| upgrade-haproxy | passed |
| init-deploy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| security-context | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-proxysql | passed |
| limits | passed |
| monitoring-2-0 | passed |
| smart-update | passed |
| demand-backup | passed |
| affinity | passed |
| self-healing-chaos | passed |
| one-pod | passed |
| auto-tuning | passed |
| self-healing-advanced | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| pitr | passed |
| self-healing-advanced-chaos | passed |
| upgrade-consistency | passed |
| operator-self-healing | passed |
| demand-backup-encrypted-with-tls | passed |
| operator-self-healing-chaos | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/881/commits/1b213b9bef54026c42d1924dcd66b6f5a2c1798a
image: `perconalab/percona-xtradb-cluster-operator:PR-881-1b213b9b`

**Review by delgod — APPROVED** [review]

File: e2e-tests/operator-self-healing/run [hunk 1]
```diff
@@ -7,6 +7,11 @@ test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 cluster="no-limits"
 
+if version_gt "1.20"; then
+	echo "Kubernetes version is 1.20+ so skipping this test because of pumba support."
```
Comment (hors) [inline e2e-tests/operator-self-healing/run:11]: @tplavcic  why do we need to have pumba tests and chaos_mesh? Maybe we can use chaos_mesh only.

**Review by hors — CHANGES_REQUESTED** [review]

File: e2e-tests/operator-self-healing/run (hunk 1 above)
Comment (delgod) [reply e2e-tests/operator-self-healing/run:11]: I believe the pumba `pause` test still has sense, so I have asked to left it

Comment (hors) [reply e2e-tests/operator-self-healing/run:11]: ok, LGTM 
@tplavcic  please update all needed jobs  in Percona-Lab/jenkins-pipelines repo too

**Pull request merged**